### PR TITLE
Fix git push error for protected CLA branch

### PR DESCRIPTION
**Pull request by gewarren**
The pull request is created from master636891325775760819 to master to fix git push error for protected CLA branch

### CODE REVIEW

**Comment by PRMerger17** [comment]
@gewarren : Thanks for your contribution! The author, @heaths, has been notified to review your proposed change.

File: docs/install/tools-for-managing-visual-studio-instances.md [hunk 1]
```diff
@@ -33,25 +33,31 @@ In addition, the [Setup Configuration API](<xref:Microsoft.VisualStudio.Setup.Co
 
 ## Using vswhere.exe
 
-`vswhere.exe` is automatically included in Visual Studio 2017 version 15.2 or later, or you can download it from [the releases page](https://github.com/Microsoft/vswhere/releases). Use `vswhere -?` to get help information about the tool. As an example, this command shows all releases of Visual Studio, including earlier versions of the product and prereleases, and outputs the results in JSON format:
+`vswhere.exe` is automatically included in Visual Studio (starting with Visual Studio 2017 version 15.2 and later versions), or you can download it from [the VSWhere releases page](https://github.com/Microsoft/vswhere/releases). Use `vswhere -?` to get help information about the tool. As an example, this command shows all releases of Visual Studio, including earlier versions of the product and prereleases, and outputs the results in JSON format:
```
Comment (heaths) [inline docs/install/tools-for-managing-visual-studio-instances.md]: We never capitalize "vswhere". This is inconsistent with existing branding.

Comment (gewarren) [reply docs/install/tools-for-managing-visual-studio-instances.md]: @TerryGLee Can you fix the capitalization here?

Comment (TerryGLee) [reply docs/install/tools-for-managing-visual-studio-instances.md]: Yes; I have PR #[4378](https://github.com/MicrosoftDocs/visualstudio-docs-pr/pull/4378) in flight.

**Pull request merged**